annotate_celltypes_condensed_sdrf.pl

```diff
@@ -55,7 +55,7 @@
   my ($expAccAux, $arrayDesign, $cell, $attributeType, $type, $value, $uri) = split /\t/, $line;
   $expAcc = $expAccAux;
   # we only care about cell types
-  $unique_properties->{ $type }->{ $value }->{ $cell } = 1 if $type =~ /.*cell type$/;
+  $unique_properties->{ $type }->{ $value }->{ $cell } = 1 if $type eq $args->{"cell_type_field"};
```

Comment:
are we running here on each cell type field per run?

Reply:
Yes. I thought of passing a comma-separated string or something in for multiple fields, but decided this was simpler. But we can go that way too.

Reply:
I wonder if we should warn if `$unique_properties` is empty, or if all the runs of unique properties are empty, so that we can spot more easily if we are missing stuff...

Reply:
well, I guess that this is all superseded by the new design that we discussed on slack.